package.json

```diff
@@ -20,20 +20,25 @@
   },
   "homepage": "https://github.com/o2r-project/o2r-substituter#readme",
   "dependencies": {
+    "adm-zip": "^0.4.7",
     "backoff": "^2.5.0",
     "body-parser": "^1.15.2",
+    "clone": "^2.1.1",
     "compression": "^1.6.2",
     "connect-mongodb-session": "^1.3.0",
     "debug": "^2.3.3",
+    "dockerode": "^2.5.0",
```

Comment:
Please check if you still need `dockerode`. Also suggest to run [`ncu`](https://github.com/tjunnone/npm-check-updates) while you're at it to have the latest working versions in the PR.